pyvolt-dpsim-demo/README.md

```diff
@@ -2,6 +2,9 @@
 
 ## Preliminaries
 
+Follow the instructions here to get started:
+https://sogno-platform.github.io/docs/getting-started/
```

Comment:
I did not have Kubernetes installed and am pretty unfamiliar with the ecosystem, so I was confused with the instructions until I followed the getting started guide.

Reply:
Good point